database/migrations/2024_02_09_300000_create_authors_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('authors', function (Blueprint $table) {
+            $table->id();
+            $table->string('first_name');
+            $table->string('last_name')->nullable();
+            $table->unsignedTinyInteger('is_individual')->default(1);
```

Comment:
I would use `->boolean('is_individual')->default(false)` here I think

Reply:
Yep we could do that too, though boolean is a synonym of tinyint in the background so I got used to that. But I'll implement boolean.